ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -210,6 +211,9 @@ class DatasetToolArgs:
     private: bool = simple_parsing.field(default=False)
     token: Optional[str] = None
 
+    # Chunk processing parameters
+    max_samples_per_chunk: int = simple_parsing.field(default=100000)
```

Comment:
I'm choosing max_samples isntead of max_size because our processing failures are usually correlated with sample # not sample size.